### ISSUE TITLE
Privileged users notice

### ISSUE THREAD

**Issue by Oaphi** [HenryEcker/Retiring-SF-SU-Draft#2]
The proposed privileged users notice includes the following line:

> You can edit the question so it’s [on-topic](/help/on-topic) for Stack Overflow

I am not sure whether it should be included as the SF/SU closure usually means it is unlikely the question can be edited to be on-topic on Stack Overflow without completely rewriting the post (which is likely against the editing guidelines). 

Would love to hear what others think on whether it should be mentioned in the notice.

**Comment by HenryEcker** [comment]
I took this from the current SU/SF guidance.

SU: We don’t allow questions about general computing hardware and software on Stack Overflow. You can edit the question so it’s [on-topic](/help/on-topic) for Stack Overflow.

SF: We don’t allow questions about professional server or networking-related infrastructure administration on Stack Overflow. You can edit the question so it’s [on-topic](/help/on-topic) for Stack Overflow.

---

I think the intention with this text is that sometimes things are on topic on Stack Overflow when performing a task _programmatically_. How to convert a single word document to a pdf using Microsoft Word would be "General Computing" while batch converting a folder of word documents to pdfs programmatically would be on topic. 

An edit which clarified (emphasised or drew attention to) the programmatic aspect _would_ be permissible. Though I agree in general Privileged users cannot fix questions closed with this reason.


This guidance is "designed to help [privileged users] know how to guide the asker in improving their question or inform them when the question should be reopened." I don't know what would be better there...

**Comment by Oaphi** [comment]
Maybe formulate it something like "If you believe the post can be made on-topic on Stack Overflow, you can edit it"? This emphasises that it is a judgement call on the part of the privileged user as well as implies that the edit should be substantial enough to ensure it is on-topic.

**Comment by HenryEcker** [comment]
I'm not convinced that 3k+ users are frequently editing off-topic questions to make them on topic. Though perhaps I am wrong. Regardless, I think it should remain as is (or fix literally every other reason)
- Recommendations: "You can edit the question so it can be answered with facts and citations."
- NDD: "Edit the question to include [desired behavior, a specific problem or error, and the shortest code necessary to reproduce the problem](/help/minimal-reproducible-example)" <- also not something someone other than OP could do
- SF/SU listed above...

---

 A few sites just say it's off topic and don't provide _any_ edit guidance to privileged users:
- Super User: 
    - This question is not about computer hardware or software, within the scope defined [in the help center](http://superuser.com/help/on-topic).
- Mathematics: 
    - This question is not about mathematics, within the scope defined in the [help center](http://math.stackexchange.com/help/on-topic).
- Tex
     - This question does not fall within the scope of TeX-LaTeX Stack Exchange as defined in the [help center](http://tex.stackexchange.com/help). These reasons may include the use of out of date packages/distributions, bug reports or simple typing errors. The comments on the question should describe the main reason.
- Electrical Engingeering:
    -  Questions on the *use* of electronic devices are off-topic as this site is intended specifically for questions on electronics *design*.

OTOH Statistical Analysis provides advice on what types of questions to ask the OP to help _them_ make their question on-topic:
- If the question is actually a statistical topic disguised as a coding question, then OP should edit the question to clarify this. After the statistical content has been clarified, the question is eligible for reopening.
- In edge cases or for very opaque questions, it can be helpful to ask what OP's **statistical** content of the question is, which might prompt OP to revise.


---


@Oaphi what about something along the lines of:
> This question does not appear to be about [a specific programming problem, software development, or tools commonly used by programmers](https://stackoverflow.com/help/on-topic). If the question is on-topic on [another site in the Stack Exchange Network](https://stackexchange.com/sites) you can leave a comment indicating where the question may get answers.

**Comment by Oaphi** [comment]
Re: how often users edit such posts. I started creating a SEDE [query](https://data.stackexchange.com/stackoverflow/query/1594024/questions-closed-as-off-topic-that-were-edited-by-other-users-afterwards) that could answer that question, but it seems as if it will be blocked by the inability to tell individual community-specific reasons apart (not exposed in SEDE). Even without the restriction to 3K+ users as editors, the query returns only about 18K posts, which is quite minuscule.

I would up for removing the guidance on editing altogether as it is unlikely that the post can be made on-topic by a simple edit (at least that is how I feel). I like your idea regarding providing guidance on *commenting* instead - let's go for it then?

**Comment by HenryEcker** [comment]
@Oaphi Thank you for looking into the query for me. Yeah, community specific reasons have always been hard to parse.

I think I've ended up with:

| Field | Text | Markdown Length |
|:---|:---|:---|
| Privileged user guidance | This question does not appear to be about a specific programming problem, software development, or tools commonly used by programmers. If you believe the question would be on-topic  on [another site in the Stack Exchange Network](https://stackexchange.com/sites) you can leave a comment to explain where the question may be able to get answers. | 344 |

[Image Mock for Privileged User Guidance][1]

This is based loosely on [one of MSE's reasons](https://data.stackexchange.com/meta.stackexchange/query/1594040):
> If you believe this question does relate to multiple network sites, help by editing the question to improve it or leave a comment explaining to the post author how to improve their question. If you believe this question would be a high-quality fit on the relevant per-site meta, please leave a comment to explain or flag for moderator attention and ask for this question to be migrated there.

Though I dropped the flag part because I don't think we need _more_ of those kind of flags on SO.


What do you think?

  [1]: https://github.com/HenryEcker/Retiring-SF-SU-Draft/blob/ddddf21f87e6d222dc6767f38a69b08b44152b76/builders/img_output/mock-privileged-banner-rendered.png

**Comment by HenryEcker** [comment]
Here's also a longer option if that's better:

| Field | Text | Markdown Length |
|:---|:---|:---|
| Privileged user guidance | If you believe the question is about [a specific programming problem, software algorithm, or tools commonly used by programmers](https://stackoverflow.com//help/on-topic) help by editing the question to improve it or leave a comment explaining to the post author how to improve their question. If the question would be on-topic  on [another site in the Stack Exchange Network](https://stackexchange.com/sites) you can leave a comment to explain where the question may be able to get answers. | 465 |

**Comment by Oaphi** [comment]
Looks good to me, @HenryEcker, with one minor nitpick as I believe it should be "on the network", not "in". Other than that - LGTM. I think it's better than giving users false hope that they can fix the post all by themselves.

**Comment by HenryEcker** [comment]
I agree. I'm going to tentatively call this the final draft.
